### PR TITLE
Correct documentation typographic erros and apply black on files

### PR DESCRIPTION
**Pull request by Paul-Saves**

### CODE REVIEW

**Comment by Paul-Saves** [comment]
KPLS is still false !

File: doc/_src_docs/kpls.rstx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+Usage
+-----
+
+.. embed-test-print-plot :: smt.surrogate_models.tests.test_surrogate_model_examples , Test , test_kpls , 80
+
+Options
+-------
+
+.. embed-options-table :: smt.surrogate_models , KPLS , options
```
Comment (relf) [inline doc/_src_docs/kpls.rstx:27]: I suppose you have not changed the whole file... Do you know why it appears has newly added?  (same thing for krg.rstx)

Comment (Paul-Saves) [reply doc/_src_docs/kpls.rstx:27]: It is because the file is in the wrong folder, sorry

Comment (Paul-Saves) [reply doc/_src_docs/kpls.rstx]: Thank you for the proof-reading :) 
The files were misplaced at the root of the documentation

**Review by relf — APPROVED** [review]

**Pull request merged**